tests/_utils.py

```diff
@@ -63,3 +64,18 @@ def load_module(location):
 
 def is_callable(obj):
     return hasattr(obj, "__call__")
+
+
+def create_guides(image):
+    mask = torch.stack(
+        (
+            torch.ones(
+                [1, 1, ceil(image.size()[2] / 2), image.size()[3]], dtype=torch.bool
+            ),
+            torch.zeros(
+                [1, 1, ceil(image.size()[2] / 2), image.size()[3]], dtype=torch.bool
+            ),
+        ),
+        2,
+    )
+    return {"top": mask, "bottom": ~mask}
```

Comment:
`pystiche` expects guides to be `torch.flaot32`:

```suggestion
    return {"top": mask.float(), "bottom": ~mask.float()}
```